prep_data.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# start elasticsearch if it's not already running
```

Comment:
this would not try to create the index again. it would just startup ES if it's not already running. i'd like the ability to run the scripts independent of each other so can't make assumptions here.